### PR TITLE
Added Cargo Commander

### PR DESCRIPTION
**Pull request by simonhyll**
Having finally reached >2000 downloads on crates.io I would love for my project to be listed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -529,6 +529,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
   * [cargo-benchcmp](https://crates.io/crates/cargo-benchcmp) — A utility to compare Rust micro-benchmarks
   * [cargo-bitbake](https://crates.io/crates/cargo-bitbake) — A cargo extension that can generate BitBake recipes utilizing the classes from meta-rust
   * [cargo-cache](https://crates.io/crates/cargo-cache) — inspect/manage/clean your cargo cache (`~/.cargo/`/`${CARGO_HOME}`), print sizes etc [![Build Status](https://github.com/matthiaskrgr/cargo-cache/workflows/ci/badge.svg?branch=master)](https://github.com/matthiaskrgr/cargo-cache/actions)
+  * [cargo-commander](https://crates.io/crates/cargo-commander) — A subcommand for `cargo` to run CLI commands similar to how the scripts section in `packag.json` works [![Build and test](https://github.com/simonhyll/cargo-commander/actions/workflows/build.yml/badge.svg)](https://github.com/simonhyll/cargo-commander/actions/workflows/build.yml)
```
Comment (palfrey) [inline README.md:532]: `packag.json` should be `package.json`. Also there's a sort ordering fix needed (see build logs)

**Comment by simonhyll** [comment]
There I think that should be it :)

**Comment by simonhyll** [comment]
As far as I can tell the check that failed had some networking issue and might just need a restart?

**Comment by palfrey** [comment]
> As far as I can tell the check that failed had some networking issue and might just need a restart?

The networking issues are that Github limits sharply how many un-authed requests you can make, and we can't sensibly share the auth tokens with PR branches. OTOH, most of the time the cache from `main` solves this, so I can just delete the cache for this branch and that gets pulled in.

That, and an actual failure on main due to a link moving, [which is now fixed](https://github.com/rust-unofficial/awesome-rust/commit/dea114c2f64c51e62e1262a4f68bf87820f5db82).

**Pull request merged**